code/modules/shuttle/computers/dropship_computer.dm

```diff
@@ -41,6 +46,8 @@
 /obj/structure/machinery/computer/shuttle/dropship/flight/proc/get_landing_zones()
 	. = list()
 	for(var/obj/docking_port/stationary/marine_dropship/dock in SSshuttle.stationary)
+		if(dock.faction != faction)
+			continue
```

Comment:
I would add a var check here for override, rather than hard coding it being impossible to share LZs